tailscale/client.go

```diff
@@ -383,6 +384,15 @@ type (
 		Connectors []string `json:"connectors,omitempty" hujson:"Connectors,omitempty"`
 		Domains    []string `json:"domains,omitempty" hujson:"Domains,omitempty"`
 	}
+
+	Grant struct {
+		Src []string              `json:"src,omitempty" hujson:"Src,omitempty"`
+		Dst []string              `json:"dst,omitempty" hujson:"Dst,omitempty"`
+		IP  []string              `json:"ip,omitempty" hujson:"IP,omitempty"`
+		App map[string][]GrantApp `json:"app,omitempty" hujson:"App,omitempty"`
```

Comment:
Optional: `App` is of type `tailcfg.PeerCapMap` (with `tailcfg` package from the main `tailscale` OSS repo)